spec/models/volunteer_spec.rb

```diff
@@ -68,6 +68,24 @@
       case_contacts.each { |c| c.reload }
       expect(case_contacts).to all(satisfy { |c| !c.active })
     end
+
+    context "when volunteer has previously been assigned a supervisor", runme: true do
```

Comment:
what this `runme` does?

Reply:
My bad! It's a tag I use during development to selectively run tests. I'll get rid of it!